app/Http/Controllers/VoteController.php

```diff
@@ -210,4 +212,36 @@ protected function setFiltersParameters(Request $request)
         $tagIds = $request->get('tag_ids');
         $this->tagIds = ($tagIds) ? explode(',', $tagIds) : [];
     }
+
+    /**
+     * Display the specific vote all results
+     * @param $voteId
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function getUserVoteResult($voteId)
+    {
+        $vote = Vote::findOrFail($voteId);
+
+        if(!$vote){
```

Comment:
I'm not sure that this checking is required. The findOrFail should call an exception if vote can't be found